Bottom: bb6a4a91788f313bb6ced2847544624141348af8
Top:    9585a623d9b2c7d92a72c1e0ba15786c2387cf88
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-24 17:56:19 +0100

Refresh of misc-c-add-query-function-ispa

---

diff --git a/src/misc.h b/src/misc.h
index 0fa6278..cd7eb3d 100644
--- a/src/misc.h
+++ b/src/misc.h
@@ -47,7 +47,7 @@ bool IsDataEmpty (DataTypePtr);
 bool IsLayerEmpty (LayerTypePtr);
 bool IsLayerNumEmpty (int);
 bool IsLayerGroupEmpty (int);
-bool IsPasteLayerEmpty (int);
+bool IsPasteEmpty (int);
 BoxTypePtr GetDataBoundingBox (DataTypePtr);
 void CenterDisplay (LocationType, LocationType, bool);
 void SetFontInfo (FontTypePtr);
